silverback/_cli.py

```diff
@@ -546,8 +548,19 @@ def create_payment_stream(
     sm = platform.get_stream_manager(network.chain_id)
     product = configuration.get_product_code(account.address, cluster.id)
 
+    accepted_tokens = platform.get_accepted_tokens(network.chain_id)
+    if token:
+        try:
+            convert(token, AddressType)
+            token_symbol = Contract(token).symbol()
+            token = accepted_tokens.get(token_symbol)
+        except ConversionError:
+            token = accepted_tokens.get(token)
```

Comment:
I think this will do what I was thinking it could do
```suggestion
        except ConversionError:
            token_symbol = token
        finally:
            token = accepted_tokens.get(token_symbol)
```